### ISSUE TITLE
Global effect Bug

### ISSUE THREAD

**Issue by Alex-Well** [SmartlyDressedGames/Unturned-3.x-Community#2320]
Found a very strange bug due to which the effect of underwater bubbles is displayed on land.
I made an effect based on custom weather, which is destroyed early with the help of the grenade script. After that, the custom weather effect itself stops working again, but instead the effect of underwater bubbles is attached to the camera.
![Снимок экрана (277)](https://user-images.githubusercontent.com/52251537/104257918-901fed00-548f-11eb-8526-e0bbda55d511.png)

**Comment by SDGNelson** [comment]
Effect based on custom weather? It sounds like the bug is related to custom weather, how are you triggering it?

**Comment by SDGNelson** [comment]
So your custom weather is destroying itself? Why?

**Comment by Alex-Well** [comment]
I used a script from a grenade to pre-destroy custom weather. My task was to create something like a trigger for my system. Later, I refused to first destroy the custom weather effect, since calling the effect again did not work. (It looks like Unturned creates a custom weather effect only in one instance, and does not hit a new one when it is called again) I found a different way to end the weather (/ weather 0) and so I just report this bug.
By the way, it would probably be useful for the modder to adjust the duration of the custom weather effect.
